src/cloudflare/internal/vectorize-api.ts

```diff
@@ -1,7 +1,14 @@
 // Copyright (c) 2023 Cloudflare, Inc.
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
-import { default as flags } from 'workerd:compatibility-flags';
+declare global {
```

Comment:
Can we move this to internal/globals.d.ts file and include it in Cloudflare/tsconfig so we don't have to import it on every place

Reply:
done